src/Comparer/StubValueComparer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Diff\Comparer;
+
+/**
+ * @since 3.0
+ *
+ * @license GPL-2.0+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class StubValueComparer implements ValueComparer {
+
+	private $returnValue;
```

Comment:
It's the return value of the stubs one public method. Given you just see it inside of the class and when constructing (and thus know its a stub), this is a good name to me. But feel free to change.